doc/admin-guide/jsonrpc/index.en.rst

```diff
@@ -0,0 +1,1692 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../../common.defs
+
+.. highlight:: cpp
+.. default-domain:: cpp
+
+
+
+.. |RPC| replace:: ``JSONRPC 2.0``
+
+.. _JSONRPC: https://www.jsonrpc.org/specification
+.. _JSON: https://www.json.org/json-en.html
+
+.. |str| replace:: ``string``
+.. |arraynum| replace:: ``array[number]``
+.. |arraynumstr| replace:: ``array[number|string]``
+.. |arraystr| replace:: ``array[string]``
+.. |num| replace:: *number*
+.. |strnum| replace:: *string|number*
+.. |object| replace:: *object*
+.. |array| replace:: *array*
+.. |optional| replace:: ``optional``
+
+.. |arrayrecord| replace:: ``array[record]``
+.. |arrayerror| replace:: ``array[errors]``
+
+.. _admin-jsonrpc:
+
+JSONRPC API
+***********
+
+
+
```

Comment:
Do multiple blank lines mean something in ReStructured Text?  Or are you just missing the wide open spaces of The Pampas?